packages/aws-cdk-lib/aws-codebuild/lib/project.ts

```diff
@@ -1715,9 +1750,15 @@ interface LinuxBuildImageProps {
 }
 
 // Keep around to resolve a circular dependency until removing deprecated ARM image constants from LinuxBuildImage
-// eslint-disable-next-line no-duplicate-imports, import/order
+// eslint-disable-next-line import/order
 import { LinuxArmBuildImage } from './linux-arm-build-image';
 
+// LinuxArmLambdaBuildImage and LinuxLambdaBuildImage reference ComputeType, so these imports should be placed below ComputeType.
+/* eslint-disable import/order */
+import { LinuxArmLambdaBuildImage } from './linux-arm-lambda-build-image';
+import { LinuxLambdaBuildImage } from './linux-lambda-build-image';
+/* eslint-enable import/order */
```

Comment:
I don't love this actually. I think the right thing to do here is to pull `ComputeType` out into its own file. Then we can import normally.